website/static/js/nodesPrivacy.js

```diff
@@ -0,0 +1,272 @@
+/**
+ * Controller for changing privacy settings for a node and its children.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+
+var oop = require('./oop');
+var $osf = require('./osfHelpers');
+var Paginator = require('./paginator');
+var osfHelpers = require('js/osfHelpers');
+var $ = require('jquery');
+var m = require('mithril');
+var Treebeard = require('treebeard');
+var $osf = require('js/osfHelpers');
```

Comment:
Same as above, `osfHelpers.js` is already required a few lines up so this can be removed.